packages/tasks/src/model-libraries.ts

```diff
@@ -831,6 +831,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path:"model_vae_fp16.pt"`,
 		snippets: snippets.threedtopia_xl,
 	},
+	anemoi: {
+		prettyLabel: "AnemoI",
+		repoName: "anemoi",
+		repoUrl: "https://github.com/ecmwf/anemoi-inference",
+		docsUrl: "https://anemoi-inference.readthedocs.io/en/latest/",
+		filter: false,
+		countDownloads: `path_filename:"model" OR path_extension:"ckpt"`,
```

Comment:
```suggestion
		countDownloads: `path_extension:"ckpt"`,
```

For what I've seen under https://huggingface.co/ecmwf/aifs-single/tree/main, it seems there's only a .ckpt file but not a "model" one.  

If you are planning to release multiple models, I'd recommend harmonizing things among them.

Reply:
Given all our models will use `.ckpt`, that makes sense